### PR TITLE
New version of instructions to support HSICC during a drupal deployment

### PR DESCRIPTION
**Pull request by eeeady**
Adds an updated set of instructions for what to do when we support HSICC during a drupal deployment.

### CODE REVIEW

File: docs/runbooks/how-to-support-eclkc-drupal-deploy.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    backend default {
+        #.host = "127.0.0.1";
+        #.port = "81";
+        .host = "127.0.0.1"; # lifeboat
+        .port = "80"; # lifeboat
+    }
+    ```
+
+2. Cycle the Varnish servers in the production loadbalancer target group.
+   1. Log into the AWS console and navigate to the [production loadbalancer configuration](https://console.aws.amazon.com/ec2/v2/home?region=us-east-1#LoadBalancers:search=ECLKC-Reader;sort=loadBalancerName).
```
Comment (kilbergr) [inline docs/runbooks/how-to-support-eclkc-drupal-deploy.md:53]: Sorry--why do we navigate here?

File: docs/runbooks/how-to-support-eclkc-drupal-deploy.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+   2. Deregister Varnish1 from the [production loadbalancer target group](https://console.aws.amazon.com/ec2/home?region=us-east-1#TargetGroup:targetGroupArn=arn:aws:elasticloadbalancing:us-east-1:802093990117:targetgroup/ECLKC-Reader-HTTPS/203a044ad376dddf) to drain the connections.
+   3. Restart the Varnish service on Varnish1 with the following command: `sudo systemctl restart varnish`.
```
Comment (kilbergr) [inline docs/runbooks/how-to-support-eclkc-drupal-deploy.md:55]: This is -- ssh into Varnish1 server and restart with that command, right?

File: docs/runbooks/how-to-support-eclkc-drupal-deploy.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+        .port = "81";
+        #.host = "127.0.0.1"; # lifeboat
+        #.port = "80"; # lifeboat
+    }
+    ```
+
+4. Restart the Varnish service on Varnish1 with the following command: `sudo systemctl restart varnish`.
+5. Reregister Varnish1 back to the [production loadbalancer target group](https://console.aws.amazon.com/ec2/home?region=us-east-1#TargetGroup:targetGroupArn=arn:aws:elasticloadbalancing:us-east-1:802093990117:targetgroup/ECLKC-Reader-HTTPS/203a044ad376dddf). Wait until the status is `Healthy`.
+6. Notify HSICC that traffic is routed to production.
+7. Trigger an update to 404.html pages on Varnish1 and Varnish2 using the script /etc/varnish/update-404.sh script.
```
Comment (kilbergr) [inline docs/runbooks/how-to-support-eclkc-drupal-deploy.md:103]:
```suggestion
7. Trigger an update to 404.html pages on Varnish1 and Varnish2 by ssh'ing into each server and running the script in /etc/varnish/update-404.sh on those machines.
```

**Review by kilbergr — COMMENTED** [review]
Added some clarifications. Can you check and make sure I'm understanding?

File: docs/runbooks/how-to-support-eclkc-drupal-deploy.md (hunk 1 above)
Comment (eeeady) [reply docs/runbooks/how-to-support-eclkc-drupal-deploy.md:53]: Good point... I don't know

File: docs/runbooks/how-to-support-eclkc-drupal-deploy.md (hunk 2 above)
Comment (eeeady) [reply docs/runbooks/how-to-support-eclkc-drupal-deploy.md:55]: yes.

**Review by kilbergr — APPROVED** [review]
THANK YOU

**Pull request merged**